docs/smart/clarityRef.md

```diff
@@ -9,141 +9,126 @@ This file contains the reference for the Clarity language.
 * TOC
 {:toc}
 
-## Block Properties
-
-The `get-block-info` function fetches property details for a block at a specified block height. For example:
-
-```cl
-(get-block-info time 10) ;; Returns 1557860301
+## Clarity Type System
+
+The Clarity language uses a strong static type system. Function arguments
+and database schemas require specified types, and use of types is checked
+during contract launch. The type system does _not_ have a universal
+super type. The type system contains the following types:
+
+* `(tuple (key-name-0 key-type-0) (key-name-1 key-type-1) ...)` -
```

Comment:
Do we want to mention implicit tuple syntax?

Reply:
I'd prefer not to here -- I want to keep this as close as possible to the SIP and with minimal surface area for now. Once I can setup a better structure for executing automatic tests for any code in the documentation, I'd be happier to expand examples.